src/models/IssueModel.js

```diff
@@ -0,0 +1,21 @@
+import {observable} from 'mobx';
+
+export default class IssueModel {
+  store;
+  issue_id;
+	name;
+
+  constructor(store, {issue_id, name}) {
+    this.store = store;
+    this.issue_id = issue_id;
+    this.name = name;
+
+    this.getId = () => {
+      return this.issue_id;
+    }
+
+    this.getName = () => {
```

Comment:
I haven't decided if I should mark this as `@computed` yet. I doubt this info would change due to user input, so probably not.